server/game/gamesteps/challenge/challengeflow.js

```diff
@@ -17,6 +17,7 @@ class ChallengeFlow extends BaseStep {
             new SimpleStep(this.game, () => this.announceChallenge()),
             new SimpleStep(this.game, () => this.promptForAttackers()),
             new SimpleStep(this.game, () => this.announceAttackerStrength()),
+            new SimpleStep(this.game, () => this.chooseStealthTargets()),
```

Comment:
Split this up to get the pipeline to continue and apply effects in between the steps